providers/os/resources/packages/windows_packages.go

```diff
@@ -452,6 +452,12 @@ func getPackageFromRegistryKeyItems(children []registry.RegistryKeyItem, platfor
 		return nil
 	}
 
+	if displayName == "" || displayVersion == "" {
```

Comment:
why do we have empty packages in the first place?

Reply:
From a bit of quick research it seems to be somewhat common, partial installs, bad uninstalls, or even some software just not putting a DisplayName in the registry during the setup/install process. 

I can debug the registry code where we pull display value and then actually check the path that's failing to confirm.

Reply:
I just want to find out if we need to fix something in cnquery. We should mark this as TODO, since generally we should not have empty package names in the first place.